mobile/src/main/java/org/openhab/habdroid/ui/OpenHABPreferencesActivity.java

```diff
@@ -139,6 +142,27 @@ public boolean onPreferenceClick(Preference preference) {
                     return true;
                 }
             });
+
+            clearCachePreference.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
+                @Override
+                public boolean onPreferenceClick(Preference preference) {
+                    if (getActivity() == null) {
```

Comment:
I don't think activity being null is a condition that actually can happen here. How should a click on a detached fragment happen?

Reply:
I'm a bit paranoid here, I'll remove it :D